nir/src/main/scala/scala/scalanative/nir/Ops.scala

```diff
@@ -84,6 +84,26 @@ sealed abstract class Op {
     case _ =>
       false
   }
+
+  final def isCommutative: Boolean = this match {
+    case Op.Bin(bin, _, _, _) =>
+      import Bin._
+      bin match {
+        case Iadd | Imul | And | Or | Xor =>
+          true
+        case Fadd | Isub | Fsub | Fmul | Sdiv | Udiv | Fdiv | Srem | Urem | Frem |
```

Comment:
If `Fmul` represents floating-point multiplication, it's commutative. (It's not associative, though.)

Reply:
👍

Reply:
`Fadd` is commutative as well it seems.